### ISSUE TITLE
The crash during deleting a hidden folder from an external flash drive

### ISSUE THREAD

**Issue by nervgh** [MeanEYE/Sunflower#194]
I use the Ubuntu 15.10.
I have an external flash drive. The file system of this drive is FAT32

![screenshot_1](https://docs.google.com/uc?authuser=0&id=0B2CP8LcWJSNSUGZ5VmJub3NMVVU&export=download)

When I attempt to remove a hidden folder from that drive, the Sunflower crashes and closes.
![screenshot_2](https://docs.google.com/uc?authuser=0&id=0B2CP8LcWJSNSY1paQk5EZmtKQmM&export=download)

**Comment by MeanEYE** [comment]
Is there a message being reported when you run Sunflower from the terminal? Also, do you have enabled delete to trash?

**Comment by nervgh** [comment]
> Also, do you have enabled delete to trash?

Yes, I have. I think the problem may be the _infinity loop_. Because I remove the hidden folder (**.Trash-1000**) then already is the **Trash**.

> Is there a message being reported when you run Sunflower from the terminal?
>> Failed running with python2.7

**Comment by MeanEYE** [comment]
Can you use `Shift+Delete` or just disable delete to trash option and see if this bug can be reproduced then? Thanks!

**Comment by nervgh** [comment]
When I use the `Shift+Delete` keyboard combination then deleting works fine.

**Comment by MeanEYE** [comment]
Okay, so removal works as expected but we don't handle exceptions occurring those operations. Thanks. Now we know where the problem is.

**Comment by cmallwitz** [comment]
I tried to reproduce this on latest develop branch:
- Sunflower has 'Delete items to trash can' and 'Show hidden files' enabled
- insert an empty flash drive, it gets auto-mounted on /media/christian/DISK_IMG
- initially there is no .Trash-1000 directory
- create empty test.txt file and delete it (by selecting file and pressing Delete key)
- test.txt gets moved to /media/christian/DISK_IMG/.Trash-1000/files/
- select .Trash-1000 and press Delete key
- "Deletion" (move to wastebasket) fails - I'm asked to try removing instead - I accept by pressing 'Retry' button and .Trash-1000 directory is completely removed (flash drive is empty afterwards)

Unless I missed something above that seems to work as expected ie. I can't reproduce issue. Any more ideas?

Thanks
Christian

**Comment by cmallwitz** [comment]
I might have a local modification that prevents this from failing but the above steps should be sufficient for reproducing the issue, correct?

**Comment by nervgh** [comment]
@cmallwitz , thanks for comments. Please, also try to reproduce [that](https://github.com/MeanEYE/Sunflower/issues/197) bug because `duplicate` tag :smile:

**Comment by cmallwitz** [comment]
Should that be closed?
